src/userSurvey.js

```diff
@@ -144,11 +172,6 @@ function setPopupToNoPromptPage() {
  * platform (e.g., SurveyMonkey, Typeform, Qualtrics, etc.).
  */
 export async function setSurvey(options) {
-    if(createdSurvey) {
-        throw new Error("userSurvey only supports one survey at present.");
-    }
-    createdSurvey = true;
-
```

Comment:
Proposed logic here:
* If there's a call to `setSurvey` with the same survey name as the survey in storage, ignore the function parameters, load all the survey attributes from storage, and continue with the survey. That allows studies with just one survey to have one call to `setSurvey` and be done.
* If there's a call to `setSurvey` with a different survey name from the survey in storage, throw an error, because two surveys are now set at the same time.
* If there's a call to `setSurvey` and there's no survey in storage, save the parameters in storage and carry out the survey.

Reply:
I implemented this, but slightly confused by what your first bullet meant. Studies will still need to call `setSurvey` upon every restart of the extension regardless of if survey attributes can be loaded from storage.